hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -516,6 +516,28 @@
         } // f: status
       ]
     } // R: FLASH_STATUS
+    {
+      name: "JEDEC_CC"
+      desc: '''JEDEC Continuous Code configuration register.
+
+        Read JEDEC ID must return continuous code if the manufacturer ID is not
+        shown in the first page of JEDEC table. This register controls the
+        Continuous Code.
+        '''
+      swaccess: "rw"
+      hwaccess: "hro"
+      fields: [
+        { bits: "15:8"
+          name: "num_cc"
+          desc: "The number that Continuous Code repeats"
+        } // f: num_cc
+        { bits: "7:0"
+          name: "cc"
+          desc: "Continuous Code byte"
+          resval: "0x7F"
```

Comment:
For what it's worth, I don't think the continuation code needs to be a configurable value--0x7F is specified in a standard.

However, it doesn't hurt anything to have the feature.

Reply:
yeah. I think `7Fh` value won't change soon (or never)  but @cfrantz wanted :)